### ISSUE TITLE
Does source filename support wildcards?

### ISSUE THREAD

**Issue by PaulMEdwards** [openalm/Extension-UtilitiesPack#93]
The documentation doesn't mention whether the source filename parameter supports wildcards... So I figured I'd ask.

If it's not currently implemented in that way, could you treat this as a feature request?

Thanks in advance.

**Comment by harshil93** [comment]
No, it doesn't support wildcards. I am accepting pull requests as I don't have time to contribute to this repo right now. Do send PRs if you like.
